robosat/tools/train.py

```diff
@@ -143,9 +144,9 @@ def train(loader, num_classes, device, net, optimizer, criterion):
         running_loss += loss.item()
 
         for mask, output in zip(masks, outputs):
-            mask = mask.data.cpu().numpy()
-            prediction = output.data.max(0)[1].cpu().numpy()
-            iou.add(mask.ravel(), prediction.ravel())
+            prediction = output.detach()
+            prediction.requires_grad = False
```

Comment:
Why detach and disabling gradients here? How is this different than moving the data to numpy arrays? Why do we need this?

Reply:
It's not a loss, it's a metrics so why would you use grads there ?

And it's different from numpy, because it's on GPU and not on CPU.
So no need to move data from one device to an another.

Reply:
Hm the docs say the result of detach will never require gradient. Do we need to explicitly set the requires grad attribute to false here then?

https://pytorch.org/docs/stable/autograd.html#torch.Tensor.detach

Reply:
You're right. No need to set these already implicit statement.
Fixed as bfeab0e

Thanks for you code review !